nx-X11/programs/Xserver/hw/nxagent/X11/include/Xrender_nxagent.h

```diff
@@ -16,7 +16,7 @@
  * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS, IN NO EVENT SHALL SuSE
```

Comment:
Do we really still need this file? It's a copy of `libXrender`'s `Xrender.h` file, which should normally be directly includable.